src/python/WMCore/MicroService/Unified/MSManager.py

```diff
@@ -114,6 +113,11 @@ def transferor(self, reqStatus):
         For references see
         https://github.com/dmwm/WMCore/wiki/ReqMgr2-MicroService-Transferor
         """
+        if not self.statusTrans:
+            firstSleep = randint(30, 90)
+            self.logger.info("MSTransferor thread initialized. Sleeping %d secs before its 1st cycle",
+                             firstSleep)
+            sleep(firstSleep)
```

Comment:
I really doubt it will solve the problem, in fact it will introduce a problem when time interval will exceed provided range and service will fail to run. I provide my comments in https://github.com/dmwm/WMCore/issues/9491 and I think the issue should be treated differently.

It should be no firstSleep at all. Instead both Transferor and Monitor should be able to *ping* an external service before the start-up. If service is not responding the code should introduce incremental sleep with possibility to give up if external service is not responding after certain iterations (which can be configurable parameter). The initial interval should be specified in configuration, instead of hardcoded here.